### PR TITLE
Best papers announcement

### PR DESCRIPTION
**Pull request by trevorcohn**

### CODE REVIEW

File: src/pages/blog/2020-11-19-best-papers.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Visually Grounded Compound PCFGs  
+*Yanpeng Zhao and Ivan Titov*
+
+**Best Short Paper**
+
+Not awarded.
+
+**Best Demonstration Paper**
+
+:hugging_face: Transformers: State-of-the-art Natural Language Processing
```
Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md]: is this `:hugging_face:` supposed to be an emoji or this literal string @trevorcohn ?

Comment (trevorcohn) [reply src/pages/blog/2020-11-19-best-papers.md]: Emoji, ideally. But either works -- as a string it would be capitalised HuggingFace.

Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md]:
```suggestion
 🤗 Transformers: State-of-the-art Natural Language Processing  
```

File: src/pages/blog/2020-11-19-best-papers.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+   and short and long format papers.
+ - The short-listed papers were sent to a [Best Paper committee](https://2020.emnlp.org/organizers/best-paper-committee) 
+   for further review, from which the overall top papers were awarded honourable mentions
+   and best paper.
+    
+Note that the best demonstration paper was decided differently, being chosen directly by the Demonstration chairs.
+
+The following papers have been given awards.
+
+**Best Paper**
```
Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md]:
```suggestion
### Best Paper
```

File: src/pages/blog/2020-11-19-best-papers.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+Digital voicing of Silent Speech  
+*David Gaddy and Dan Klein*
+
+**Honourable Mention Papers**
```
Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md]:
```suggestion
### Honourable Mention Papers
```

File: src/pages/blog/2020-11-19-best-papers.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+GLUCOSE: GeneraLized and COntextualized Story Explanations  
+*Nasrin Mostafazadeh, Aditya Kalyanpur, Lori Moon, David Buchanan, Lauren Berkowitz, Or Biran and Jennifer Chu-Carroll*
+
+Spot The Bot: A Robust and Efficient Framework for the Evaluation of Conversational Dialogue Systems  
+*Jan Deriu, Don Tuggener, Pius von Däniken, Jon Ander Campos, Alvaro Rodrigo, Thiziri Belkacem, Aitor Soroa, Eneko Agirre and Mark Cieliebak*
+
+Visually Grounded Compound PCFGs  
+*Yanpeng Zhao and Ivan Titov*
+
+**Best Short Paper**
```
Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md]:
```suggestion
### Best Short Paper
```

File: src/pages/blog/2020-11-19-best-papers.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+Not awarded.
+
+**Best Demonstration Paper**
```
Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md]:
```suggestion
### Best Demonstration Paper
```

File: src/pages/blog/2020-11-19-best-papers.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+   and best paper.
+    
+Note that the best demonstration paper was decided differently, being chosen directly by the Demonstration chairs.
+
+The following papers have been given awards.
+
+### Best Paper
+
+Digital voicing of Silent Speech  
+*David Gaddy and Dan Klein*
```
Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md:32]:
```suggestion
- Digital voicing of Silent Speech  
*David Gaddy and Dan Klein*
```

File: src/pages/blog/2020-11-19-best-papers.md [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+*Clara Meister, Ryan Cotterell and Tim Vieira*
+
+GLUCOSE: GeneraLized and COntextualized Story Explanations  
+*Nasrin Mostafazadeh, Aditya Kalyanpur, Lori Moon, David Buchanan, Lauren Berkowitz, Or Biran and Jennifer Chu-Carroll*
+
+Spot The Bot: A Robust and Efficient Framework for the Evaluation of Conversational Dialogue Systems  
+*Jan Deriu, Don Tuggener, Pius von Däniken, Jon Ander Campos, Alvaro Rodrigo, Thiziri Belkacem, Aitor Soroa, Eneko Agirre and Mark Cieliebak*
+
+Visually Grounded Compound PCFGs  
+*Yanpeng Zhao and Ivan Titov*
```
Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md:46]:
```suggestion
- If beam search is the answer, what was the question?  
*Clara Meister, Ryan Cotterell and Tim Vieira*

- GLUCOSE: GeneraLized and COntextualized Story Explanations  
*Nasrin Mostafazadeh, Aditya Kalyanpur, Lori Moon, David Buchanan, Lauren Berkowitz, Or Biran and Jennifer Chu-Carroll*

- Spot The Bot: A Robust and Efficient Framework for the Evaluation of Conversational Dialogue Systems  
*Jan Deriu, Don Tuggener, Pius von Däniken, Jon Ander Campos, Alvaro Rodrigo, Thiziri Belkacem, Aitor Soroa, Eneko Agirre and Mark Cieliebak*

- Visually Grounded Compound PCFGs  
*Yanpeng Zhao and Ivan Titov*
```

File: src/pages/blog/2020-11-19-best-papers.md [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+
+### Best Short Paper
+
+Not awarded.
+
+### Best Demonstration Paper
+
+ 🤗 Transformers: State-of-the-art Natural Language Processing
```
Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md:54]:
```suggestion
-  🤗 Transformers: State-of-the-art Natural Language Processing  
```

File: src/pages/blog/2020-11-19-best-papers.md (hunk 7 above)
Comment (admackin) [inline src/pages/blog/2020-11-19-best-papers.md]:
```suggestion
If beam search is the answer, what was the question?  
*Clara Meister, Ryan Cotterell and Tim Vieira*

<p/>

GLUCOSE: GeneraLized and COntextualized Story Explanations  
*Nasrin Mostafazadeh, Aditya Kalyanpur, Lori Moon, David Buchanan, Lauren Berkowitz, Or Biran and Jennifer Chu-Carroll*

<p/>

Spot The Bot: A Robust and Efficient Framework for the Evaluation of Conversational Dialogue Systems  
*Jan Deriu, Don Tuggener, Pius von Däniken, Jon Ander Campos, Alvaro Rodrigo, Thiziri Belkacem, Aitor Soroa, Eneko Agirre and Mark Cieliebak*

<p/>

Visually Grounded Compound PCFGs  
*Yanpeng Zhao and Ivan Titov*
```

**Pull request merged**